src/Commands/MinifyApplicationCommand.php

```diff
@@ -61,4 +62,50 @@ protected function cleanUpEnvFile(string $appPath): void
 
         file_put_contents($envFile, $envValues);
     }
+
+    protected function removeIgnoredFilesAndFolders(string $appPath): void
+    {
+        $this->info('Cleaning up ignored files and folders…');
+
+        $itemsToRemove = config('nativephp.cleanup_exclude_files', []);
+
+        foreach ($itemsToRemove as $item) {
+            $fullPath = $appPath.'/'.$item;
+
+            if (file_exists($fullPath)) {
+                if (is_dir($fullPath)) {
+                    $this->delete_directory_recursive($fullPath);
+                } else {
+                    array_map('unlink', glob($fullPath));
+                }
+            } else {
+                foreach (glob($item) as $pathFound) {
+                    unlink($pathFound);
+                }
+            }
+        }
+    }
+
+    private function delete_directory_recursive($dir)
```

Comment:
suggest use CamelCase and type in params

Reply:
This is now done.